### ISSUE TITLE
Remove old rake tasks for ingesting works and ETDs

### ISSUE THREAD

**Issue by kerchner** [gwu-libraries/scholarspace-hyrax#571]
- Double check that nobody is using these any more.
- Also consider archiving https://github.com/gwu-libraries/etd-loader and https://github.com/gwu-libraries/batch-loader

They will always be in the github history if needed!

**Comment by kerchner** [comment]
@dolsysmith What are your thoughts on the above?  I assume that any ScholComm (or other Librarian) users loading new Works, or collections of Works, have been doing this through the front end and/or with your assistance.

**Comment by dolsysmith** [comment]
I would agree. Bulkrax supersedes these methods. At least, I haven't used the batch-loader in years.

**Comment by kerchner** [comment]
@dolsysmith @kilahimm I have archived https://github.com/gwu-libraries/batch-loader and https://github.com/gwu-libraries/etd-loader

I'll proceed with removing the obsolete rake tasks (and perhaps address #541 at the same time)
